python/py-gpilab-core/Portfile

```diff
@@ -8,8 +8,8 @@ github.setup            gpilab core-nodes 2.2.0 v
 name                    py-gpilab-core
 # Bump to rebuild with latest py-gpilab-framework
 revision                0
-python.versions         36 37 38
-python.default_version  37
+python.versions         36 37 38 39
+python.default_version  38
```

Comment:
Any reason to not make 3.9 the default at this time ?

Reply:
Yes, there is a (binary only) "Philips Data Reader" that they make available; they don't have a py39 version of it  available yet.

Reply:
I do need to drop the 36 flavors, however, with h5py dropping 36 support...